provider/resources.go

```diff
@@ -138,3 +141,21 @@ func Provider() tfbridge.ProviderInfo {
 
 	return prov
 }
+
+func editRules(defaults []info.DocsEdit) []info.DocsEdit {
+	textReplace := func(old, new string) info.DocsEdit {
+		o, n := []byte(old), []byte(new)
+		return info.DocsEdit{
+			Path: "*",
+			Edit: func(_ string, content []byte) ([]byte, error) {
+				return bytes.ReplaceAll(content, o, n), nil
+			},
+		}
+	}
+	return append(defaults,
+		textReplace("## Importing", "## Import"),
+		textReplace(`"terraform-`, `"pulumi-`),
+		textReplace("\n[docs-import]: https://www.terraform.io/docs/import/index.html\n", "\n"),
```

Comment:
This might be a little too specific wrt the newline:

<img width="648" alt="Screenshot 2024-09-30 at 11 13 29 AM" src="https://github.com/user-attachments/assets/7ea84956-929c-4113-ba13-3dd140f80b10">

Reply:
It's unrelated to the newline thing, but I can fix it.